profiles/static/style.css

```diff
@@ -1,27 +1,27 @@
 .card-custom-wide-img {
-  height: 130px;
-  background-image: url("/static/assets/images/diamonds-light.png");
-  transition: all 0.5s ease;
+    height: 130px;
+    background-image: url("/static/assets/images/diamonds-light.png");
+    transition: all 0.5s ease;
```

Comment:
Please don't reindent _everything_

Reply:
Reverted changes to indentation, only changed originally to add uniformity.